src/Domain/ElementId.hpp

```diff
@@ -29,12 +33,15 @@ class ElementId {
   /// Create the ElementId of the root Element of a Block.
   explicit ElementId(size_t block_id) noexcept;
 
+  /// Convert an ElementIndex to an ElementId
+  // clang-tidy: mark explicit: we want to allow conversion
+  ElementId(const ElementIndex<VolumeDim>& index) noexcept;  // NOLINT
```

Comment:
What is the intended use for this as an implicit conversion?

Reply:
Since a function inside an Action might take an ElementId, but the passed in index in the Charm++ array an implicit conversion would simplify calling such functions. Do you think that's a bad idea?